Dockerfile

```diff
@@ -1,21 +1,15 @@
 FROM ubuntu:18.04
 ADD https://raw.githubusercontent.com/wireshark/wireshark/master/tools/debian-setup.sh /
-RUN apt update \
-	&& apt install gnupg apt-utils -y \
-	&& echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" > /etc/apt/sources.list.d/clang7.list \
-	&& echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" > /etc/apt/sources.list.d/clang8.list \
+RUN apt-get update \
```

Comment:
I know that apt gives that creepy message back, but I've been using apt in scripts for years it is actually stable.

Reply:
It is still not promised to be stable and some subcommands behave a bit differently. apt-get is the right command to use in scripts.